pkg/webhook/pod/pods.go

```diff
@@ -44,17 +45,24 @@ type PodAdmissionControl struct {
 	pdControl pdapi.PDControlInterface
 	// the map of the service account from the request which should be checked by webhook
 	serviceAccounts sets.String
+	// recorder to send event
+	recorder record.EventRecorder
 }
 
 const (
 	stsControllerServiceAccounts = "system:serviceaccount:kube-system:statefulset-controller"
+	podDeleteMsgPattern          = "pod[%s] deleted"
```

Comment:
Add namespace?

Reply:
I think the namespace here is not necessary as the event belongs to a certain tidbcluster.

Reply:
Then add tc info?
Is the tc info already shown in the event?